doc/howtos/software-update-server.rst

```diff
@@ -0,0 +1,61 @@
+.. _software-update-server:
+
+Software Update: Building an Ostro |trade| OS  Repository
+#########################################################
+
+This technical note provides instructions supporting the creation
```

Comment:
This technical note provides instructions for creating a repository used for updating devices based on Ostro OS.